src/sentry/utils/sdk_crashes/sdk_crash_detection.py

```diff
@@ -13,8 +17,11 @@ class SDKCrashReporter:
     def __init__(self):
         self
 
-    def report(self, event: Event) -> None:
-        return
+    def report(self, event_data: Dict[str, Any], event_project_id: int) -> Event:
+        from sentry.event_manager import EventManager
+
+        manager = EventManager(event_data)
```

Comment:
Not sure if this is necessary, but better safe than sorry:

```suggestion
        manager = EventManager(event_data)
        manager.normalize()
```